.github/workflows/build-triton-wheel.yml

```diff
@@ -54,12 +54,12 @@ jobs:
             docker-image: "pytorch/manylinux2_28-builder:cpu"
         include:
           - device: "rocm"
-            rocm_version: "6.2"
+            rocm_version: "6.2.4"
           - device: "cuda"
             rocm_version: ""
     timeout-minutes: 40
     env:
-      DOCKER_IMAGE: ${{ matrix.device == 'rocm' && format('pytorch/manylinux-builder:rocm{0}', matrix.rocm_version) || matrix.docker-image }}
+      DOCKER_IMAGE: ${{ matrix.device == 'rocm' && format('pytorch/manylinux2_28-builder:rocm{0}', matrix.rocm_version) || matrix.docker-image }}
```

Comment:
Please separate triton rocm build from Whl binaries build. Otherwise the Smoke test will fail.

Reply:
@atalman Actually, smoke tests succeeded in one of the earlier commits in this PR. Can you please point out which specific smoke test you think would fail?